docs/data-cookbooks/throttling-queued-executions.md

```diff
@@ -83,12 +80,9 @@ Follow the instructions to [re-deploy your Cumulus application](./../deployment/
 
 ```json
 {
-  "meta": {
-    "queues": {
-      "backgroundJobQueue": "<backgroundJobQueue_SQS_URL>"
-    },
+  "cumulus_meta": {
     "queueExecutionLimits": {
-      "backgroundJobQueue": 5
+      "<backgroundJobQueue_SQS_URL>": 5
```

Comment:
I need to consider this a bit more --`cumulus_meta` as defined here: https://nasa.github.io/cumulus/docs/workflows/cumulus-task-message-flow is not supposed to be modified *in* the workflow.   We're assuming that contract by convention, with no explicit validation and this would change that contract.

I realize this is just explicitly exposing configuration of values therein in a way that was already technically possible, but going from <don't do that> to <usually don't do that> might have subtle integration usability consequences

Reply:
Ignore this entirely ^